contracts/test/4337/Test4337ModuleAndHandler.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.7.0 <0.9.0;
+pragma abicoder v2;
+
+import "../../libraries/SafeStorage.sol";
+import "./UserOperation.sol";
+
+interface ISafe {
+    function execTransactionFromModule(address to, uint256 value, bytes memory data, uint8 operation) external returns (bool success);
+}
+
+/// @dev A Dummy 4337 Module/Handler for testing purposes
+/// @dev DO NOT USE IN PRODUCTION
```

Comment:
Should we add a little bit more contexd? I.e. this module doesn't perform any checks (neither on that the interaction comes from the entrypoint, nor on the userop)

Reply:
Improved the explanation a little bit